assets/Bookreader logo - cool grad.svg

```diff
@@ -0,0 +1 @@
2.85869-2.426559-5.4585l-1.011963-2.41346c-.180069-.43287-.365799-.88093-.557123-1.34402l-1.214558-2.95759c-.213441-.52255-.432345-1.05979-.656648-1.61156l-1.410062-3.48361c-.245632-.60921-.496529-1.23262-.752628-1.87005l-1.598478-3.99153-1.72015-4.3201-1.83867-4.64061-1.95404-4.95308-2.06626-5.25751-3.302906-8.43946-3.537676-9.07917-3.761811-9.69173-3.975311-10.27713-4.178178-10.83538-2.892904-7.52032-4.492656-11.70547-4.667165-12.19132-4.83104-12.65003-4.984279-13.08157-5.126885-13.48598-5.258857-13.86322-7.198718-19.024202-5.525435-14.637884-5.62196-14.924631-5.707849-15.184225-5.783106-15.41667-5.847726-15.621964-3.929135-10.515287-5.931797-15.903785-5.968062-16.036677-5.993691-16.142418-4.004782-10.806949-6.012775-16.258318-6.010047-16.291656c-.667237-1.811031-1.334227-3.622176-2.000904-5.433268l-2.951567-8.02244c-1.927823-5.24264-3.854584-10.487817-5.780201-15.735335l-5.773337-15.749373c-.961623-2.625999-1.92294-5.252534-2.883939-7.879579l-5.762107-15.768203-5.753999-15.77927c-.958296-2.630734-1.916234-5.261881-2.873803-7.893414l-5.740903-15.793643c-1.912103-5.265963-3.822648-10.533275-5.731551-15.801739l-5.7217-15.808647-5.711353-15.814366c-2.853006-7.908415-5.701946-15.818529-8.546539-23.729673l-5.683306-15.823465c-.946225-2.637407-1.891945-5.274879-2.837151-7.912392l-5.664987-15.825358c-1.886217-5.275149-3.770295-10.55026-5.65215-15.825136l-5.638818-15.823724-5.624986-15.821123-5.610658-15.817336c-2.801664-7.907522-5.597769-15.813178-8.388033-23.716299l-5.57266-15.802665c-2.782354-7.899444-5.558682-15.795907-8.328704-23.688722l-5.531552-15.780565-5.514238-15.769646c-.917569-2.627299-1.834396-5.254092-2.75047-7.880357l-5.487334-15.751038c-2.739069-7.872146-5.471179-15.739083-8.19605-23.600142l-5.440006-15.714083c-.905031-2.617642-1.809237-5.234581-2.712608-7.850793l-5.410116-15.688345c-.899988-2.613203-1.799119-5.22563-2.697384-7.837255l-5.379107-15.659932c-2.684272-7.824959-5.360557-15.642257-8.028578-23.451227l-5.324937-15.606635c-.885625-2.599189-1.770311-5.197404-2.654048-7.794618l-5.2909422-15.571093-5.2676574-15.545911c-3.5038992-10.355283-6.9919433-20.692986-10.4634687-31.011525l-5.1948162-15.463237c-6.9096819-20.597956-13.7519622-41.116085-20.5215334-61.54171l-5.0634625-15.301677c-11.7825484-35.6625162-23.3383612-71.0320328-34.6389925-106.0406081l-.8339544-2.585837.472606-.1062199.5311782-.1168008 1.0475959-.2204466c.5188885-.1058431 1.0304238-.2051206 1.5346546-.2978537l.9987367-.1767466.9793448-.1593325c1.2929037-.2008508 2.5343268-.3553649 3.7251941-.4639448l.8836857-.0728285c.1457059-.0107047.2906255-.0206933.4347605-.0299665l.8554106-.0470612c10.7163206-.4875326 16.9221406 3.1478002 19.4103281 10.5609279l.14362.4359964 405.8186381 1154.9518069c.985053 2.80344 3.632712 4.67947 6.604177 4.67947h77.769362c2.939849 0 5.566447-1.83692 6.5751-4.59832l421.876101-1154.9749929c4.069466-10.8507358 13.347819-14.4447201 29.507364-10.2992503l1.00835.2662957c.169526.0460292.339787.0928819.510785.1405589z" fill="#194880"/></g></g></g></svg>
```

Comment:
These are a big different since they're not used inside BR itself, but let's place in `src/assets` for consistency :+1:

Reply:
ok sounds good! that I can do